safe/report/test/test_earthquake_report.py

```diff
@@ -105,7 +105,8 @@ def test_earthquake_population_without_aggregation(self):
         """:type: safe.report.report_metadata.Jinja2ComponentsMetadata"""
 
         expected_context = {
-            'table_header': u'Estimated Number of people',
+            'table_header': u'Estimated Number of people affected '
+                            u'per MMI intensity',
```

Comment:
We usually pull left when we can. You can put bracket after table_header and then pull left.